src/install.js

```diff
@@ -166,11 +168,18 @@ async function main() {
     'remoteresources3decrypt': { install: argv.rrs3d || argv['remoteresources3decrypt'], uri: `${fileSource}/RemoteResourceS3Decrypt/${filePath}` },
     'mustachetemplate': { install: argv.mtp || argv['mustachetemplate'], uri: `${fileSource}/MustacheTemplate/${filePath}` },
     'featureflagsetld': { install: argv.ffsld || argv['featureflagsetld'], uri: `${fileSource}/FeatureFlagSetLD/${filePath}` },
-    'encryptedresource': { install: argv.er || argv['encryptedresource'], uri: `${fileSource}/EncryptedResource/${filePath}` },
     'managedset': { install: argv.ms || argv['managedset'], uri: `${fileSource}/ManagedSet/${filePath}` },
     'impersonationwebhook': { install: argv.iw || argv['impersonationwebhook'], uri: `${fileSource}/ImpersonationWebhook/${filePath}` }
   };
 
+  // Handle deprecated resources
+  for( const resourceName of Object.getOwnPropertyNames( resourcesObj ) ) {
+    if( Object.hasOwn( deprecatedResourcesObj, resourceName ) ) {
+      delete resourcesObj[ resourceName ];
+    }
+  }
```

Comment:
Would it be more efficient to do this way?
```suggestion
  // Handle deprecated resources
  for( const resourceName of Object.getOwnPropertyNames( deprecatedResourcesObj ) ) {
    if( Object.hasOwn(resourcesObj , resourceName) ) {
      delete resourcesObj[ resourceName ];
    }
  }
```